src/reason_parser.mly

```diff
@@ -1053,6 +1054,8 @@ conflicts.
           NEW NATIVEINT PREFIXOP STRING TRUE UIDENT
           LBRACKETPERCENT
 
+%left VARIABLEINFIXOP
```

Comment:
I believe it should be right by the other infix ops (you have to decide if it's higher or lower precedence than `INFIXOP4` etc.